log_util.c

```diff
@@ -160,7 +160,7 @@ int logBacktrace(int level, struct __sourceloc whence)
     close(stdout_fds[0]);
     // Need the (void*) cast on Solaris because it defines NULL as 0L and gcc doesn't accept it as a
     // sentinal
-    execlp("gdb", "gdb", "-n", "-batch", "-x", tempfile, execpath, pidstr, (void*)NULL);
+    execlp("sudo", "sudo", "gdb", "-n", "-batch", "-x", tempfile, "-p", pidstr, (void*)NULL);
```

Comment:
As this is really needed only for testing/ debugging configuration is a good option. An additional help text if gdb startup fails could describe the configuration options of the environment variable and the handling.

To reduce the possible vulnerability implications I would stick with having just an option in the environment variable not the whole command line.